contrib/default.config

```diff
@@ -31,33 +30,13 @@ ST_STBOOT_VERSION=6e2173d
 # ST_LINUXBOOT_KERNEL_VERSION defines the linux kernel version of the LinuxBoot
 # distribution.
 #
-ST_LINUXBOOT_KERNEL_VERSION=5.10.21
+ST_LINUXBOOT_KERNEL_VERSION=5.15.15
```

Comment:
I think we can bump another minor release while we're at it (current as of writing this is 5.15.18)